source/cloud-security/gcp/prerequisites/dependencies.rst

```diff
@@ -5,73 +5,22 @@
 Installing dependencies
 =======================
 
-The Wazuh module for Google Cloud Pub/Sub and the Wazuh module for Google Cloud Storage buckets can be configured on both the Wazuh server and the monitored endpoint. This choice depends on where you want to reach the Google Cloud services. Perform the steps below to install Python and Pip on the endpoint you are performing the integration.
+.. |service| replace:: GCP
 
-.. warning::
-   
-   The Wazuh server includes all dependencies installed; the following steps are only necessary when configuring the modules on a monitored endpoint.
+.. include:: /_templates/cloud/notes.rst
 
 Python
 ------
 
-The Wazuh module for Google Cloud Pub/Sub and the Wazuh module for Google Cloud Storage buckets require `Python 3 <https://www.python.org/>`__. It's compatible with
-`Python |PYTHON_CLOUD_CONTAINERS_MIN|–|PYTHON_CLOUD_CONTAINERS_MAX| <https://www.python.org/downloads/>`_. While later Python versions should work as well, we can't assure they are compatible.
-
-Run the following command to install Python 3 if you do not already have it installed:
-
-.. tabs::
-
-   .. group-tab:: Yum
-
-      .. code-block:: console
-
-         $ sudo yum update && yum install python3
-
-   .. group-tab:: APT
-
-      .. code-block:: console
-
-         $ sudo apt-get update && apt-get install python3
-
-Run the command below to install Pip, the Python package manager. Most UNIX distributions have this tool available in their software repositories by default.
-
-Perform the next steps if you do not already have it installed.
-
-.. tabs::
-
-   .. group-tab:: Yum
-
-      .. code-block:: console
+.. |service_module| replace:: ``GCP``
```

Comment:
We could remove this replacement and simply use `|service|` if they get unified.